app/assets/stylesheets/osem-rating.css.scss

```diff
@@ -1,19 +1,3 @@
-/* Styling for voting on proposals*/
-.myrating.bright { background-image: image-url("star-bright.png"); }
-.myrating.glow { background-image: image-url("star-glow.png"); }
-.othersrating.bright { background-image: image-url("star-bright.png"); }
-.avgrating.bright { background-image: image-url("star-bright.png"); }
-.avgrating {
-  background: image-url("star.png") 0 0;
-  margin-right: -2px;
-  width: 24px;
-  height: 24px;
-  display: inline-block;
+img.raty-cancel {
```

Comment:
The cancel button doesn't cancel the rating but instead clicking on cancel changes the rating to '0' stars. This property is to prevent the cancel button from being displayed.